### ISSUE TITLE
WordPress testing official SQLite Support. Useful?

### ISSUE THREAD

**Issue by xet7** [sandstormports/wordpress-sandstorm#44]
@ocdtrekkie @zenhack 

Would this be useful for Sandstorm WordPress?

https://github.com/WordPress/performance/pull/547

https://news.ycombinator.com/item?id=34020786

**Comment by xet7** [comment]
It seems to use some MySQL to SQLite abstraction:
https://github.com/aaemnnosttv/wp-sqlite-db

**Comment by ocdtrekkie** [comment]
Yeah, if WordPress picks up official SQLite support it would simplify our version a lot, and plugin compatibility would improve too.
